ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -449,41 +440,28 @@ banalPointSchedule blockTree =
 -- The LoE is only resolved when all peers with forks at that block have been disconnected from,
 -- in particular due to a decision based on the Genesis density criterion.
 --
--- This is implemented by initializing the schedule to contain only the honest chain, then folding
--- over the adversaries and inserting ten ticks before each honest tick.
--- It only makes sense when there's a single adversary – otherwise, each subsequent adversary will
--- be ten times as fast as the previous one.
---
--- It uses 'banalStates' to convert each fragment to a schedule (which advances by one block per
--- tick).
-fastAdversaryPointSchedule ::
-  BlockTree TestBlock ->
-  Maybe PointSchedule
-fastAdversaryPointSchedule blockTree =
-  peer2Point frags (foldGenPeers frags trans)
+-- This is implemented by initializing each peer's schedule with 'banalStates' (which advances by
+-- one block per tick) and assigning interval lengths to each peer tick based on the frequency
+-- config in the first argument, then sorting the resulting absolute times.
+frequencyPointSchedule :: Peers Int -> BlockTree TestBlock -> Maybe PointSchedule
```

Comment:
The `freqs` parameter can use more of an explanation. How does the integer of each peer affect the resulting point schedule?